blueprint/templates/subcommands.go

```diff
@@ -159,7 +167,7 @@ func SubCommands() []cli.Command {
 				},
 				cli.StringFlag{
 					Name:  "parameter-values",
-					Usage: "A map that assigns a value to each script parameter. Example: '{\"param1\":\"val1\",\"param2\":\"val2\"}'",
+					Usage: "A list of comma separated script parameters paired by Id and Value, i.e: --parameter-values param1:val1,param2:val2",
```

Comment:
What if the value I want to indicate contains a comma?

Reply:
good point! do you have any proposal? ... because not sure about to use another character or  a complex syntax. Maybe it should be restored to original json format?

Reply:
No other character is going to solve the problem, and escaping it is going to be too complex both to implement and to educate users... I think we should go back to json and maybe have a -from-file alternative, as we do for configuration attributes

Reply:
Done. restored default JSON option; and added functionality for reading from file or stdin